src/options.cpp

```diff
@@ -98,9 +98,14 @@ static void usage()
 	printf("-keymap <keymap>\n");
 	printf("\tEnable a specific keyboard layout decode table.\n");
 
-	printf("-log {K|S|V}...\n");
-	printf("\tEnable logging of (K)eyboard, (S)peed, (V)ideo.\n");
+	printf("-log {K|S|V|Cl|Cb|Ca|Co}...\n");
```

Comment:
I understand the reasoning behind this, it makes sense to group these together if they're all "logging-like" functions. But since I'd really like the new CPU logging to only be compiled in when `TRACE` is defined to the compiler, it might make sense to move these to their own command-line option. 

I don't think the whole option would need to be `#if defined(TRACE)` like the actual trace outputting code, But it should document that it only works if running a build that compiled in the CPU trace logic.

Reply:
If you do move the flags to a new `-trace` option or something, I think that would be a reasonable place to add `d` and/or `x` to specify whether trace output should be in decimal or hex, per my other comment regarding `getShowHex()`. If neither 'd' or 'x' is specified, I would probably default to hexadecimal output.

Reply:
I've been looking into the different optionis. There are really a lot  of options in the box that have to do with "trace" or "output".

-log
-verbose
-dump

wanted to avoid to introduce an other option, as it introduced an other complexity.
Therefore, i've solved it as follows:

<img width="648" alt="image" src="https://user-images.githubusercontent.com/13690775/205430155-09408c80-436d-4b23-8070-8d4a4b360295.png">

Suggest we keep it simple :-)